test/Sentry.Tests/SentryOptionsTests.cs

```diff
@@ -50,39 +59,119 @@ public void IsTracingEnabled_Default_False()
     }
 
     [Fact]
-    public void EnableTracing_WhenNull()
+    public void IsTracingEnabled_EnableTracing_True()
     {
         var sut = new SentryOptions
         {
-            EnableTracing = null
+            EnableTracing = true
         };
 
-        Assert.Null(sut.EnableTracing);
-        Assert.Equal(0.0, sut.TracesSampleRate);
+        Assert.True(sut.IsTracingEnabled);
     }
 
     [Fact]
-    public void EnableTracing_WhenFalse()
+    public void IsTracingEnabled_EnableTracing_False()
     {
         var sut = new SentryOptions
         {
             EnableTracing = false
         };
 
-        Assert.False(sut.EnableTracing);
-        Assert.Equal(0.0, sut.TracesSampleRate);
+        Assert.False(sut.IsTracingEnabled);
     }
 
     [Fact]
-    public void EnableTracing_WhenTrue()
+    public void IsTracingEnabled_TracesSampleRate_Zero()
     {
         var sut = new SentryOptions
         {
-            EnableTracing = true
+            TracesSampleRate = 0.0
+        };
+
+        Assert.False(sut.IsTracingEnabled);
```

Comment:
It seems like the SentryOptions.EnableTracing property is redundant... tracing could be implicitly disabled by setting `SentryOptions.TracesSampleRate` to `0` and implicitly enabled by setting `SentryOptions.TracesSampleRate` to something `> 0`. 

Under the hood, maybe we don't need a backing variable for EnableTracing property? Possibly something for v4?

Reply:
I think `EnableTracing` was added later, after `TracesSampleRate` was there.

Reply:
Yes. It is specifically redundant, and done across all SDKs. Intended to be a simplified shorthand for enabling tracing without thinking about sample rate.